### DISCUSSION TITLE
Need help unenrolling a Chromebox

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#788] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I recently bought a Chromebox off of eBay. However, when turning it on, it seems to be enrolled in a school. My son tried to unenroll it with "shimmer" but he told me he couldn't "find an rma shim", which according to him means that he can't "shimmer" it (I have no idea what any of this means). He directed me here to ask you guys if there was any way I could possibly unenroll it?

The model is an Acer CXI4 Chromebox
The board name is Puff
Developer Mode is blocked
The account it asks us to sign in with says it ends with "hsd.k12.or.us"
I intend to use this as a home media center for music & videos when it's unenrolled

**Comment by AshtonDavies** [comment, marked as answer]
The official link was taken down, but it's rehosted at [dl.osu.bio](https://dl.osu.bio)

**Reply by ghost** [reply]
I'm looking there, but I don't see one called "puff". Is there any way I can unenroll this without shimmering it?

**Reply by AshtonDavies** [reply]
Enterprise enrollment glitch does exactly that

**Reply by ghost** [reply]
Will that let me enable Developer Mode?

**Reply by AshtonDavies** [reply]
I think so. If it's still blocked after bypassing enrollment, you can try Fakemurk, but it will probably work without that.
